scss/partials/_typography.scss

```diff
@@ -0,0 +1,155 @@
+.p {
+  @extend p;
+
+  &--lead,
+  &--lead-1 {
+    font-size: 2.8rem;
+    line-height: 1.286;
+  }
+
+  &--lead-emphasis,
+  &--lead-1-emphasis {
+    @extend .p--lead;
```

Comment:
Compared to other areas where I've brought up `@extend`, I think this usage is good.